### ISSUE TITLE
Can't make Indictor invisible?

### ISSUE THREAD

**Issue by der-zonk** [akinsho/bufferline.nvim#277]
Since the bar freaks out when I remove the indicator I thought I could just make it invisible by setting ctermbg/fg to none but it then falls back to some random white color..
Sorry if this is trivial but I'm just a hobby nerd and have not much experience with this stuff.
It accepts any bg color but for fg it's either a color I pick or white as fallback when set to none.
Thanks.
EDIT: Somehow github ate something I wrote.. So I found out the color falls back to 'Normal'..
Would it be possible to be not linked to Normal by default so one can change it easy?
I tried relinking BufferLineIndicator[...] to something else but the issue was then the same as it did not accept fg=NONE..

I want to create a mostly transparent vim config and this looks just bad imo if I use a bg on tabs..
![2021-12-16 22:44:00](https://user-images.githubusercontent.com/80225209/146453840-354328bc-78ae-4fae-8bc7-beac4fa0ab0f.png)
and this looks even more off if u ave some line there where it just does not need to be one..
![2021-12-16 22:45:26](https://user-images.githubusercontent.com/80225209/146453957-e759fe26-7582-4b87-9721-99f2492d61d6.png)
I tried just removing the indicatoer but then the tab names just jump arond when u switch tabs..
I would be really gratefull for any help..

**Comment by der-zonk** [comment]
As a workaround I found out u can use a blank Braille character, namely this one here:
https://www.compart.com/en/unicode/U+2800
The tabs are stable and don't jump around.
Still it would be nice if one would not have to jump through these hoops if one does not want the indicator IMHO.

**Comment by akinsho** [comment]
@der-zonk I'm not sure what character you are referring to as the indicator. I call the indicator the left aligned character that shows which tab is selected. As the README states
```
    -- NOTE: this plugin is designed with this icon in mind,
    -- and so changing this is NOT recommended, this is intended
    -- as an escape hatch for people who cannot bear it for whatever reason
    indicator_icon = '▎'
```
you can change this character but this plugin was designed with it in mind, you can just as easily change it to an empty string or space which should be equivalent to setting it to some other character 🤷🏿

**Comment by der-zonk** [comment]
> @der-zonk I'm not sure what character you are referring to as the indicator. 

I thought I wrote I mean the  BufferLineIndicator


>  you can just as easily change it to an empty string or space which should be equivalent to setting it to some other character 🤷🏿

Then why does the text in the tabs freak out when setting an empty string?

https://user-images.githubusercontent.com/80225209/146758685-60b52e42-d3f5-45d9-babb-4f5573f287e7.mp4

This does only then not happen when setting a blank character (And not any one works, at least it did not for me, I had to use the one I linked).
Equivalent for me means it has no impact on the outcome which it here definitively has.
